doc/lightning-invoice.7.md

```diff
@@ -68,6 +69,10 @@ payment.
 If specified, *cltv* sets the *min_final_cltv_expiry* for the invoice.
 Otherwise, it's set to the parameter **cltv-final**.
 
+If *deschash* is true (default false), then a hash of the *description*,
```

Comment:
```suggestion
If *deschashonly* is true (default false), then a hash of the *description*,
```
Also, the main clause, starting with `then`, is incomplete (probably missing verb and subject).

Reply:
Thanks!  Obviously I didn't finish that thought!